org.eclipse.xtext.xbase.lib/src/org/eclipse/xtext/xbase/lib/ArrayLiterals.java

```diff
@@ -13,11 +13,27 @@
  * Factory methods for arrays.
  * 
  * @author Sven Efftinge - Initial contribution and API
+ * @author Stephane Galland - Add operations on double-dimension arrays.
  * @since 2.4
  */
 @GwtCompatible
 public class ArrayLiterals {
 
+	/**
+	 * @param size0
+	 *            the first size for the array to be created
+	 * @param size1
+	 *            the second size for the array to be created
+	 * @return an array of the given sizes
+	 * @since 15.0
```

Comment:
2.15

Reply:
Oh. Sorry. I will fix this version number right now.